README.md

```diff
@@ -26,7 +26,7 @@ You can install the package globally and use it as command line tool:
 
 ```sh
 # Install the package globally
-npm i -g <%- _.pack.name %>
```

Comment:
Whouppss, this `README.md` file is generated, and you just discovered a bug. :smile: Thanks! :+1: